sorc/build_ufs.sh

```diff
@@ -27,7 +27,7 @@ done
 
 cd "${cwd}/ufs_model.fd"
 
-MAKE_OPT="-DAPP=${APP} -DCCPP_SUITES=${CCPP_SUITES}"
+MAKE_OPT="-DAPP=${APP} -D32BIT=ON -DCCPP_SUITES=${CCPP_SUITES}"
```

Comment:
The APP being MS2SWA is also part of this, although this will be changed later, see issue: https://github.com/ufs-community/ufs-weather-model/issues/1540 

This is following how the ufs-weather-model defines to use mixed-mode.  @binli2337 can you answer @WalterKolczynski-NOAA  ufs-weather-model build questions about mixed mode?

Reply:
@binli2337  -- pinging you again to point you to @WalterKolczynski-NOAA questions about details for the 32 bit compilation.  

Also note that with moving towards newer model versions, the APP is no longer MS2SWA

Reply:
The mixed-mode PRs (ufs-weather-model PR#1514 and PR#1541) only added one extra option to build the 32-bit dynamical core. The build options for other components were not changed.